### PR TITLE
SSticker minor refactor

### PR DESCRIPTION
**Pull request by AffectedArc07**
## What Does This PR Do
Cleans up SSticker and removes some old, unused features which were part of it. Also documents vars and adds some rough time gauging to things.

## Why It's Good For The Game
Cleanup of code

## Changelog
N/A

### CODE REVIEW

**Comment by AffectedArc07** [comment]
I'll refactor world/Reboot() when I do `SSblackbox`

File: code/game/world.dm [hunk 1]
```diff
@@ -104,6 +103,7 @@ GLOBAL_LIST_EMPTY(world_topic_handlers)
 		spawn(0)
 			to_chat(world, "<span class='boldannounce'>Rebooting world immediately due to host request</span>")
 		rustg_log_close_all() // Past this point, no logging procs can be used, at risk of data loss.
+		TgsReboot()
```
Comment (AffectedArc07) [inline code/game/world.dm:106]: This was done in this PR to simplify things down the line.

TLDR; TGS uses the `TgsReboot()` signal if youre on the basic watchdog to kill the DD process. In our current config, it would kill right at end round before anything had been saved if we were in basic mode. This fixes that.

File: code/controllers/subsystem/ticker.dm [hunk 2]
```diff
@@ -194,93 +226,34 @@ SUBSYSTEM_DEF(ticker)
...
+	to_chat(world, "<span class='darkmblue'><B>Enjoy the game!</B></span>")
+	world << sound('sound/AI/welcome.ogg')
+
+	if(SSholiday.holidays)
+		to_chat(world, "<span class='darkmblue'>and...</span>")
+		for(var/holidayname in SSholiday.holidays)
+			var/datum/holiday/holiday = SSholiday.holidays[holidayname]
+			to_chat(world, "<h4>[holiday.greet()]</h4>")
+	
+	SSdiscord.send2discord_simple_noadmins("**\[Info]** Round has started with no admins online")
```
Comment (AffectedArc07) [inline code/controllers/subsystem/ticker.dm:255]:
```suggestion
	SSdiscord.send2discord_simple_noadmins("**\[Info]** Round has started")
```

File: code/controllers/subsystem/ticker.dm [hunk 3]
```diff
@@ -120,43 +130,47 @@ SUBSYSTEM_DEF(ticker)
...
 		SSjobs.ResetOccupations()
-		if(!src.mode)
-			src.mode = pickweight(runnable_modes)
-		if(src.mode)
-			var/mtype = src.mode.type
-			src.mode = new mtype
+		if(!mode)
+			mode = pickweight(runnable_modes)
+		if(mode)
+			var/mtype = mode.type
```
Comment (SabreML) [inline code/controllers/subsystem/ticker.dm:156]:
```suggestion
		if(!mode)
			mode = pickweight(runnable_modes)
		else
			var/mtype = mode.type
```

Comment (AffectedArc07) [reply code/controllers/subsystem/ticker.dm:156]: Its intentionally like this. A mode can be picked on like 154 then accepted on 155. If you change this to an `else`, like 156 will never trigger.

**Comment by AffectedArc07** [comment]
DNM, need to fix some reboot stuff

**Comment by AffectedArc07** [comment]
Cannot reproduce. Declaring a fluke since everything else in the reboot chain worked.
